render_json_test.go

```diff
@@ -71,8 +77,9 @@ func TestJSONConsumeIndented(t *testing.T) {
 		IndentJSON: true,
 	})
 
+	var renErr error
```

Comment:
`renErr` is used here to differentiate between the error returned from the `JSON` render and the one returned from `json.Unmarshal` below. Using a different name allows us to ensure both are `nil`.